client/my-sites/plan-price/index.jsx

```diff
@@ -43,17 +53,27 @@ export default class PlanPrice extends Component {
 				<sup className="plan-price__fraction">
 					{ rawPrice - price.integer > 0 && price.fraction }
 				</sup>
+				{ config.isEnabled( 'show-tax' ) &&
+					taxText && (
+						<sup className="plan-price__tax-amount">
+							{ translate( '(+%(taxText)s tax)', { args: { taxText } } ) }
+						</sup>
+					) }
 			</h4>
 		);
 	}
 }
 
+export default localize( PlanPrice );
+
```

Comment:
Import `PlanPrice` into the test as mentioned, or just use `i18n.translate` 😸

Reply:
> or just use i18n.translate 😸

OMG, the simple 🤦‍♂️